LIMS2DB/objectsDB/process_categories.py

```diff
@@ -79,6 +79,8 @@
     '2205' : 'Adapter ligation and reverse transcription',
     '2254' : 'ONT End-Prep and Cleanup',
     '2360' : 'Library Preparation & Amplification',
+    '2656' : 'Deparafinization, H&E, Tissue Imaging',
```

Comment:
2656 and 2657 count as both Prepstart and Workset?

Reply:
Yes, this protocol was not properly set up in LIMS, and I have no other option to fix it. Later I will talk with someone to re-do the protocol.

Reply:
So is the PR ok to merge with this?

Reply:
Yes, for now it is OK to merge. I will talk with Alfred to change to protocol when he is back.